README.md

```diff
@@ -65,7 +65,7 @@ Examples:
 
 Tools:
 
-- tools/[argdist](tools/argdist.py): Display function parameter values as a histogram or frequency count. [Examples](tools/argdist_example.txt).
+- tools/[argdist](tools/argdist.py): Display function parameter values as a histogram or frequency count. [Examples](tools/argdist_examples.txt).
```

Comment:
@4ast They do, but OTOH if you look at all the man files it says on the bottom to look at the corresponding _examples.txt file (plural, not singular). I don't mind either way... But it's not consistent already.